REPORT.md

```diff
@@ -0,0 +1,76 @@
+# Reporting capabilities within zAppBuild
+
+zAppBuild provides a set of reporting capabilities, which are part of the build framework itself to address some common demands of mainframe development teams.
+
+Developers are particularly interested if
```

Comment:
Add a colon at the end of this line.  "Developers are particularly interested if:"